Docs17.4

josephsl opened the pull request:
### Link to issue number:
None

### Summary of the issue:
Documentation updates and edits.

### Description of how this pull request fixes the issue:

Added Macedonian as a supported language, edits to what's new document and user guide, edited URL's displayed on user guide to point to newer websites (in case of developer guide) and use HTTPS (Add-ons website).

### Testing performed:
None

### Known issues with pull request:
None

### Change log entry:
None

josephsl committed:
What's new: style, wording.

* WDAG: clarified which version of Windows 10 introduced it (Fall Creators Update).
* espeak-ng: shortened commit hash to eight characters.
* Addons -> add-ons.
* Portable copy command line: remove 'support for' to make the entry consistent.
* Braille dispaly key assignment: punctuation change.

josephsl committed:
New language: Macedonian.

josephsl committed:
MathPlayer documentation URL encased in t2t link markup for consistency.

josephsl committed:
User guide: change http to https for add-ons website, change addresses for developer Guide and extra voices wiki page.

josephsl commented:
If no objections are raised, I recommend merging this into master before translatable string freeze. Thanks,

ThomasStivers commented on user_docs/en/userGuide.t2t:
```diff
@@ -1748,14 +1748,14 @@ To access the Add-ons Manager from anywhere, please assign a custom gesture usin
 + Supported Speech Synthesizers +[SupportedSpeechSynths]
 This section contains information about the speech synthesizers supported by NVDA.
-For an even more extensive list of  free and commercial synthesizers that you can purchase and download for use with NVDA, please see the page at [http://www.nvda-project.org/wiki/ExtraVoices http://www.nvda-project.org/wiki/ExtraVoices].
+For an even more extensive list of  free and commercial synthesizers that you can purchase and download for use with NVDA, please see [extra voices page https://github.com/nvaccess/nvda/wiki/ExtraVoices].
```
Perhaps "see the extra voices page" instead of "see extra voices page".

derekriemer replied:
agree.

josephsl commented:
Hi, @DerekRiemer, any issues with grammar in this commit?

 

From: ThomasStivers [mailto:notifications@github.com] 
Sent: Sunday, October 15, 2017 11:10 AM
To: nvaccess/nvda <nvda@noreply.github.com>
Cc: Joseph Lee <joseph.lee22590@gmail.com>; Author <author@noreply.github.com>
Subject: Re: [nvaccess/nvda] Docs17.4 (#7662)

 

@ThomasStivers commented on this pull request.

  _____  

In user_docs/en/userGuide.t2t <https://github.com/nvaccess/nvda/pull/7662#discussion_r144726972> :

>  
 ++ Reload plugins ++
 This item, once activated, reloads app modules and global plugins without restarting NVDA, which can be useful for developers.
 
 + Supported Speech Synthesizers +[SupportedSpeechSynths]
 This section contains information about the speech synthesizers supported by NVDA.
-For an even more extensive list of  free and commercial synthesizers that you can purchase and download for use with NVDA, please see the page at [http://www.nvda-project.org/wiki/ExtraVoices http://www.nvda-project.org/wiki/ExtraVoices].
+For an even more extensive list of  free and commercial synthesizers that you can purchase and download for use with NVDA, please see [extra voices page https://github.com/nvaccess/nvda/wiki/ExtraVoices].

Perhaps "see the extra voices page" instead of "see extra voices page".

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/nvaccess/nvda/pull/7662#pullrequestreview-69425096> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AHgLkNmvpAUz4cWYSZLhggmy0PZoi7mRks5sskp9gaJpZM4P51EY> .

derekriemer commented on user_docs/en/changes.t2t:
```diff
@@ -12,19 +12,20 @@ What's New in NVDA
-- Version information for addons is now reported when being updated. (#5324)
-- Added support for new command line parameters to create a portable copy of NVDA. (#6329)
-- Support for Microsoft Edge running within Windows Defender Application Guard (#7600)
+- Version information for add-ons is now reported when being updated. (#5324)
```
is -> are?

derekriemer commented on user_docs/en/changes.t2t:
```diff
@@ -12,19 +12,20 @@ What's New in NVDA
-- Added support for new command line parameters to create a portable copy of NVDA. (#6329)
-- Support for Microsoft Edge running within Windows Defender Application Guard (#7600)
+- Version information for add-ons is now reported when being updated. (#5324)
+- Added new command line parameters to create a portable copy of NVDA. (#6329)
```
what did you change here?

derekriemer commented on user_docs/en/changes.t2t:
```diff
@@ -12,19 +12,20 @@ What's New in NVDA
-- Support for Microsoft Edge running within Windows Defender Application Guard (#7600)
+- Version information for add-ons is now reported when being updated. (#5324)
+- Added new command line parameters to create a portable copy of NVDA. (#6329)
+- Support for Microsoft Edge running within Windows Defender Application Guard in Windows 10 Fall Creators Update. (#7600)
```
(and above)?

derekriemer commented on user_docs/en/changes.t2t:
```diff
@@ -12,19 +12,20 @@ What's New in NVDA
+- Added new command line parameters to create a portable copy of NVDA. (#6329)
+- Support for Microsoft Edge running within Windows Defender Application Guard in Windows 10 Fall Creators Update. (#7600)
 - If running on a laptop or tablet, NVDA will now report when a charger is connected/disconnected, and when the screen orientation changes. (#4574, #4612)
+- New language: Macedonian.
```
I'd say this more flowery. Like 
NVDA has now been translated into Macedonian!

derekriemer replied:
not going to make it flowery?

derekriemer commented on user_docs/en/changes.t2t:
```diff
@@ -12,19 +12,20 @@ What's New in NVDA
 - In Browse mode for Firefox and Chrome, the name of form field groups are now announced when moving into them with quick navigation or when tabbing. (#3321)
 - In browse mode, the quick navigation command for embedded objects (o and shift+o) now includes audio and video elements as well as elements with the aria roles application and dialog. (#7239)
-- Espeak-ng has been updated (to commit 01919cd48a566cdf34347784b2e74554b376e900), this resolves some issues with producing release builds. (#7385)
+- Espeak-ng has been updated to commit 01919cd4, which resolves some issues with producing release builds. (#7385)
```
Arguably, () around the commit stuff was more correct since it lets the non-essential info be in parenthesis. However, the way you did this is more compact. If noone else objects, I think this is better. Maybe a comma before to could make it slightly more obvious the commit info is extra, but it really doesn't matter.

derekriemer commented on user_docs/en/changes.t2t:
```diff
@@ -34,7 +35,7 @@ What's New in NVDA
 - Cursor routing for the MDV Lilli Braille display no longer incorrectly moves one braille cell ahead of where it should be. (#7469)
 - In Internet Explorer and other MSHTML documents, the HTML5 required attribute is now supported to indicate the required state of a form field. (#7321)
-- Braille displays are now updated when typing Arabic characters in a left-aligned WordPad document (#511).
+- Braille displays are now updated when typing Arabic characters in a left-aligned WordPad document. (#511)
```
Nice catch.

derekriemer reviewed (changes requested):
Have you checked for any other references to nvda-project.org, community.*?

josephsl commented:
Hi, ah yes, thanks for reminding me about that. Will do that as part of a more general source code cleanup. Thanks.

 

From: Derek Riemer [mailto:notifications@github.com] 
Sent: Sunday, October 15, 2017 11:29 AM
To: nvaccess/nvda <nvda@noreply.github.com>
Cc: Joseph Lee <joseph.lee22590@gmail.com>; Author <author@noreply.github.com>
Subject: Re: [nvaccess/nvda] Docs17.4 (#7662)

 

@derekriemer requested changes on this pull request.

Have you checked for any other references to nvda-project.org, community.*?

  _____  

In user_docs/en/changes.t2t <https://github.com/nvaccess/nvda/pull/7662#discussion_r144727144> :

> @@ -12,19 +12,20 @@ What's New in NVDA
 - Initial support for Windows 10 on ARM64. (#7508)
 - Early support for reading and interactive navigation of mathematical content for Kindle books with accessible math. (#7536)
 - Added support for Azardi e-book reader. (#5848)
-- Version information for addons is now reported when being updated. (#5324)
-- Added support for new command line parameters to create a portable copy of NVDA. (#6329)
-- Support for Microsoft Edge running within Windows Defender Application Guard (#7600)
+- Version information for add-ons is now reported when being updated. (#5324)

is -> are?

  _____  

In user_docs/en/changes.t2t <https://github.com/nvaccess/nvda/pull/7662#discussion_r144727196> :

> @@ -12,19 +12,20 @@ What's New in NVDA
 - Initial support for Windows 10 on ARM64. (#7508)
 - Early support for reading and interactive navigation of mathematical content for Kindle books with accessible math. (#7536)
 - Added support for Azardi e-book reader. (#5848)
-- Version information for addons is now reported when being updated. (#5324)
-- Added support for new command line parameters to create a portable copy of NVDA. (#6329)
-- Support for Microsoft Edge running within Windows Defender Application Guard (#7600)
+- Version information for add-ons is now reported when being updated. (#5324)
+- Added new command line parameters to create a portable copy of NVDA. (#6329)

what did you change here?

  _____  

In user_docs/en/changes.t2t <https://github.com/nvaccess/nvda/pull/7662#discussion_r144727216> :

> @@ -12,19 +12,20 @@ What's New in NVDA
 - Initial support for Windows 10 on ARM64. (#7508)
 - Early support for reading and interactive navigation of mathematical content for Kindle books with accessible math. (#7536)
 - Added support for Azardi e-book reader. (#5848)
-- Version information for addons is now reported when being updated. (#5324)
-- Added support for new command line parameters to create a portable copy of NVDA. (#6329)
-- Support for Microsoft Edge running within Windows Defender Application Guard (#7600)
+- Version information for add-ons is now reported when being updated. (#5324)
+- Added new command line parameters to create a portable copy of NVDA. (#6329)
+- Support for Microsoft Edge running within Windows Defender Application Guard in Windows 10 Fall Creators Update. (#7600)

(and above)?

  _____  

In user_docs/en/changes.t2t <https://github.com/nvaccess/nvda/pull/7662#discussion_r144727244> :

>  - If running on a laptop or tablet, NVDA will now report when a charger is connected/disconnected, and when the screen orientation changes. (#4574, #4612)
+- New language: Macedonian.

I'd say this more flowery. Like
NVDA has now been translated into Macedonian!

  _____  

In user_docs/en/changes.t2t <https://github.com/nvaccess/nvda/pull/7662#discussion_r144727309> :

>  - Web dialogs in Firefox and Chrome web browsers now automatically use browse Mode, unless inside of a web application. (#4493)
 - In browse mode, tabbing and moving with quick navigation commands no longer announces jumping out of containers such as lists and tables, which makes navigating more efficient. (#2591)
 - In Browse mode for Firefox and Chrome, the name of form field groups are now announced when moving into them with quick navigation or when tabbing. (#3321)
 - In browse mode, the quick navigation command for embedded objects (o and shift+o) now includes audio and video elements as well as elements with the aria roles application and dialog. (#7239)
-- Espeak-ng has been updated (to commit 01919cd48a566cdf34347784b2e74554b376e900), this resolves some issues with producing release builds. (#7385)
+- Espeak-ng has been updated to commit 01919cd4, which resolves some issues with producing release builds. (#7385)

Arguably, () around the commit stuff was more correct since it lets the non-essential info be in parenthesis. However, the way you did this is more compact. If noone else objects, I think this is better. Maybe a comma before to could make it slightly more obvious the commit info is extra, but it really doesn't matter.

  _____  

In user_docs/en/changes.t2t <https://github.com/nvaccess/nvda/pull/7662#discussion_r144727336> :

> @@ -34,7 +35,7 @@ What's New in NVDA
 - NVDA no longer becomes noticeably slower when many tabs are open in either Firefox or Chrome web browsers. (#3138)
 - Cursor routing for the MDV Lilli Braille display no longer incorrectly moves one braille cell ahead of where it should be. (#7469)
 - In Internet Explorer and other MSHTML documents, the HTML5 required attribute is now supported to indicate the required state of a form field. (#7321)
-- Braille displays are now updated when typing Arabic characters in a left-aligned WordPad document (#511).
+- Braille displays are now updated when typing Arabic characters in a left-aligned WordPad document. (#511)

Nice catch.

  _____  

In user_docs/en/userGuide.t2t <https://github.com/nvaccess/nvda/pull/7662#discussion_r144727428> :

>  
 ++ Reload plugins ++
 This item, once activated, reloads app modules and global plugins without restarting NVDA, which can be useful for developers.
 
 + Supported Speech Synthesizers +[SupportedSpeechSynths]
 This section contains information about the speech synthesizers supported by NVDA.
-For an even more extensive list of  free and commercial synthesizers that you can purchase and download for use with NVDA, please see the page at [http://www.nvda-project.org/wiki/ExtraVoices http://www.nvda-project.org/wiki/ExtraVoices].
+For an even more extensive list of  free and commercial synthesizers that you can purchase and download for use with NVDA, please see [extra voices page https://github.com/nvaccess/nvda/wiki/ExtraVoices].

agree.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/nvaccess/nvda/pull/7662#pullrequestreview-69425236> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AHgLkIcqrip6y-nVLv6cz9omlgpHqt5Tks5ssk7zgaJpZM4P51EY> .

michaelDCurran commented:
where is this up to? translation freeze really needs to be tomorrow if possible.

josephsl committed:
Merge branch 'master' into docs17.4

# Conflicts:
#	user_docs/en/changes.t2t

josephsl committed:
Final notes

josephsl commented:
Hi, ready to go. I think you should be the one writing highlight text for 2017.4. Thanks.

 

From: Michael Curran [mailto:notifications@github.com] 
Sent: Tuesday, October 31, 2017 11:36 PM
To: nvaccess/nvda <nvda@noreply.github.com>
Cc: Joseph Lee <joseph.lee22590@gmail.com>; Author <author@noreply.github.com>
Subject: Re: [nvaccess/nvda] Docs17.4 (#7662)

 

where is this up to? translation freeze really needs to be tomorrow if possible.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/nvaccess/nvda/pull/7662#issuecomment-341002875> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AHgLkM_A1ZVUOoumNMEO0o4765vayQC_ks5syBFpgaJpZM4P51EY> .

michaelDCurran commented:
Yep, I'll right the highlight paragraph.

Thanks

derekriemer commented on user_docs/en/changes.t2t:
```diff
@@ -25,13 +26,13 @@ What's New in NVDA
 - In browse mode, the quick navigation command for embedded objects (o and shift+o) now includes audio and video elements as well as elements with the aria roles application and dialog. (#7239)
-- Espeak-ng has been updated to commit 01919cd4, which resolves some issues with producing release builds. (#7385)
+- Espeak-ng has been updated (to release 1.49.2), this resolves some issues with producing release builds. (#7385, #7583)
 - On the third activation of the 'read status bar' command, it's contents is copied to the clipboard. (#1785)
```
please fix it's to its

derekriemer reviewed (commented):
You updated espeak from master it looks like. Please make sure you don't cause an espeak regression in master before the freeze.

josephsl committed:
What's new: linguistic fixes.

josephsl committed:
Merge branch 'master' into docs17.4

josephsl committed:
What's new: capitalization

michaelDCurran reviewed (changes requested):
include/espeak has been incorrectly changed.

josephsl committed:
Merge branch 'master' into docs17.4

michaelDCurran committed:
Switch eSpeak back to 1.49.2 (NVDA master)

michaelDCurran reviewed (approved):
I corrected the eSpeak submodule commit back to eSpeak 1.49.2

josephsl commented:
Hi, thanks (too many merges from master perhaps…).

 

From: Michael Curran [mailto:notifications@github.com] 
Sent: Wednesday, November 1, 2017 7:29 PM
To: nvaccess/nvda <nvda@noreply.github.com>
Cc: Joseph Lee <joseph.lee22590@gmail.com>; Author <author@noreply.github.com>
Subject: Re: [nvaccess/nvda] Docs17.4 (#7662)

 

@michaelDCurran approved this pull request.

I corrected the eSpeak submodule commit back to eSpeak 1.49.2

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/nvaccess/nvda/pull/7662#pullrequestreview-73671751> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AHgLkNRqpY1sYHthXRr0HV6VyRIsneUmks5sySj0gaJpZM4P51EY> .
